lib/ts/station/FSM.ex

```diff
@@ -45,6 +47,7 @@ defmodule Station.FSM do
   defstate start do
     # On getting the data input, go to ready state
     defevent input_data(station_data = [_station_struct, _dependency]) do
+      Logger.debug(fn -> "Initialise the station data" end)
```

Comment:
The StationStruct has station_number and station_name. Both can be added to this log message.